python/paddle/fluid/optimizer.py

```diff
@@ -301,25 +302,45 @@ def minimize(self,
         This method combines interface `append_backward()` and
         `create_optimization_pass()` into one.
         """
-        params_grads = append_backward(loss, parameter_list, no_grad_set,
-                                       [error_clip_callback])
+        if imperative_base.enabled():
+            if parameter_list is not None:
+                params_grads = parameter_list
+            else:
```

Comment:
Can we force to pass parameter_list in imperative mode for now?

Reply:
Maybe discussed in the follow up PRs